### ISSUE TITLE
Stuck MIDI Notes

### ISSUE THREAD

**Issue by nedrob** [surge-synthesizer/surge#4013]
**Bug Description:**

Getting 80% stuck midi notes when playing from my external keyboard, 100% stuck notes if I use after touch. Using Surge 1.8 inside Ableton 11. Novation SL MKIII keyboard.

**Surge Version**
1.8

* Version:
* Plugin: VST3
* Bitness: 64-bit

**Reproduction Steps:**
1. Go to any patch
2. Play on external keyboard 
3. With or without stuck aftertouch
4. Stuck notes

**Expected Behavior:**
Anodes not stuck

**Screenshots:**
If applicable, add screenshots/GIF/videos to help explain your problem.

**Computer Information (please complete the following information):**
 - OS: Windows 10
 - Host: Ableton
 - Version: 11

**Comment by baconpaul** [comment]
In abelton 11 is the device set up as an MPE device? And if so does the problem resolve when you either turn MPE on In surge of turn Mpe off on the device?

**Comment by jomicof** [comment]
I also got stuck notes in Ableton 11, but only on some patches when Play mode was any of the Mono choices and only when the notes were played legato-style.  When 2 notes were played legato-style, the last not played got stuck (or latched?) and only released  when another note was played non-legato-style.  MPE was turned off on the device in Ableton and also off in Surge.   Turning MPE on in Ableton device did not help.  But, with MPE on in Surge (and off in Ableton device), the problem went away.   I saw the same issue with both Surge-1.8.1 and Surge-NIGHTLY-2021-03-16-6da806f.
